credentials/apps/verifiable_credentials/rest_api/v1/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class ProgramCredentialSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Comment:
Seems, we use this serializer for validation only?
If so, making it readonly should increase productivity.